BridgeCareApp/BridgeCare/Models/ValidateModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace BridgeCare.Models
+{
+    public class ValidateEquationModel
+    {
+        public string equation { get; set; }
```

Comment:
the properties are starting with small casing. Are these to match something with the client app?